website/src/pages/case-studies/nutanix-fixed-log4j-with-sourcegraph.tsx

```diff
@@ -0,0 +1,342 @@
+import React, { FunctionComponent } from 'react'
+import { PageProps, Link } from 'gatsby'
+import slugify from 'slugify'
+
+import { Blockquote, BlockquoteWithLogoTop } from '../../components/Blockquote'
+import { CaseStudyJumbotron, AuthorBio } from '../../components/content/CaseStudyPage'
+import { CaseStudyCard, CASESTUDIES } from '../../components/product/CaseStudyCard'
+import { ContentSection } from '../../components/content/ContentSection'
+import Layout from '../../components/Layout'
+
+import ClipboardTextOutlineIcon from 'mdi-react/ClipboardTextOutlineIcon'
+import ChartBarIcon from 'mdi-react/ChartBarIcon'
+import CheckCircleOutlineIcon from 'mdi-react/CheckCircleOutlineIcon'
+import FlagOutlineIcon from 'mdi-react/FlagOutlineIcon'
+
+const NutanixCaseStudy: FunctionComponent<PageProps> = props => (
+    <Layout
+        location={props.location}
+        meta={{
+            title: 'Nutanix fixed Log4j quickly and confidently with Sourcegraph',
+            description:
+                'Sourcegraph enables Nutanix to find and fix security vulnerabilities quickly across the code base. When the Log4j vulnerability rocked the industry, Nutanix was able to confidently identify every instance of Log4j across its sprwaling codebase and deliver patches to its customers that fully remediated the vulnerability within 4 days.',
+            image: 'https://about.sourcegraph.com/sourcegraph-og.png',
+        }}
+    >
+        <div className={`${slugify('Nutanix').toLowerCase()}-case-study`}>
+            <CaseStudyJumbotron
+                className="bg-gradient-onahau-fog text-black height-md-450 height-auto p-2"
+                color="white"
+                customer="Nutanix"
+            >
+                <h1 className="pt-5 pb-6 display-2 font-weight-bold mw-600 mx-auto">
+                    Nutanix fixed Log4j quickly and confidently with Sourcegraph
+                </h1>
+            </CaseStudyJumbotron>
+
+            <ContentSection color="white" className="py-6 text-center col-sm-12 col-md-9 col-lg-6">
```

Comment:
I know this is a different approach, but If you remove `col-sm-12 col-md-9 col-lg-6` and add `mw-600` it essentially does the same thing but prevents the content from expanding too far beyond the Figma artboard size.

Reply:
Just for the sake of being a bit more precise? That seems to work too 👍